### ISSUE TITLE
Training loss over 400?

### ISSUE THREAD

**Issue by rzcwade** [vrenkens/nabu#34]
Hi @vrenkens ,

I got your LAS + WSJ model training working but I got training loss over 400 to start with. What's your results like?

Thanks.

**Comment by vrenkens** [comment]
This is probably because your using alignments, check other issue for a response :)

**Comment by rzcwade** [comment]
@vrenkens , thanks for your response. I think it's because it uses the sum of cross entropy as loss function.
